### ISSUE TITLE
Is there a way to specify a timeout so I can continue with other work more quickly if the target site times out?

### ISSUE THREAD

**Issue by lorddev** [ghorsey/OpenGraph-Net#23]
**Is your feature request related to a problem? Please describe.**
I have a problem where I'm retrieving a list of open-graph responses in an API. If one site is down it breaks my whole API; I'd like to be able to provide a timeout so that if it takes more than a few seconds to provide a response I can move on to the next one in the list.

**Describe the solution you'd like**
An overload, e.g. of `OpenGraph.ParseUrlAsync(url, timeout);` might be nice.

**Comment by lorddev** [comment]
I'm trying this:
```csharp
			OpenGraph graph;
			Task<OpenGraph> task = OpenGraph.ParseUrlAsync(url);
			if (await Task.WhenAny(task, Task.Delay(5000))== task)
			{
				graph = task.Result;
			}
			else
			{
				return null;
			}
```

**Comment by ghorsey** [comment]
I will think about how this can be incorporated

**Comment by ghorsey** [comment]
I added this extra argument. It's in milliseconds. See http://ghorsey.github.io/OpenGraph-Net/api/OpenGraphNet.OpenGraph.html#OpenGraphNet_OpenGraph_ParseUrlAsync_System_String_System_String_System_Boolean_System_Int32_

**Comment by ghorsey** [comment]
@lorddev this fix is in v3.2.6
